framework/components/ecl-tables/ecl-tables.js

```diff
@@ -6,7 +6,9 @@
 
 export function eclTables(elements = null) {
   const tables =
-    elements === null ? document.getElementsByClassName('ecl-table') : elements;
+    elements === null
```

Comment:
Why not `elements || document.getElementsByClassName('ecl-table--responsive')`?

Reply:
Good point, the default value for the argument is there indeed.